assets/scss/_custom.scss

```diff
@@ -171,7 +171,7 @@ body.td-404 main .error-details {
 // Flip-Nav
 .flip-nav .td-navbar {
   background-color: white !important;
-  box-shadow: 0 1px 2px $medium-grey;
+  box-shadow: 0 1px 8px 0px #4c4c4c3d;
```

Comment:
Hi @kawaljeet99 

Would it be possible to use one of the theme's named colors here?

Reply:
Hi @sftim 

I have tweaked some values and achieved the similar look.
And as you asked, I've done this with named color from the theme.